src/main/java/edu/colorado/cires/cmg/s3out/AwsS3ClientMultipartUpload.java

```diff
@@ -79,11 +79,15 @@ private AwsS3ClientMultipartUpload(S3Client s3, ContentTypeResolver contentTypeR
   }
 
   @Override
-  public String createMultipartUpload(String bucket, String key) {
+  public String createMultipartUpload(String bucket, String key, ObjectMetadata objectMetadata) {
 
     CreateMultipartUploadRequest.Builder builder = CreateMultipartUploadRequest.builder()
-        .bucket(bucket)
-        .key(key);
+            .bucket(bucket)
+            .key(key);
+
+    if (objectMetadata != null) {
```

Comment:
Maybe move this after

`contentTypeResolver.resolveContentType(key).ifPresent(builder::contentType);`

This would allow the explicit ObjectMetadata.contentType to override the value potentially set by the ContentTypeResolver